core/src/main/java/greencity/webcontroller/ManagementAchievementController.java

```diff
@@ -0,0 +1,125 @@
+package greencity.webcontroller;
+
+import greencity.dto.PageableAdvancedDto;
+import greencity.dto.achievement.AchievementManagementDto;
+import greencity.dto.achievement.AchievementPostDto;
+import greencity.dto.achievement.AchievementVO;
+import greencity.dto.genericresponse.GenericResponseDto;
+import greencity.service.AchievementCategoryService;
+import greencity.service.AchievementService;
+import greencity.service.LanguageService;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+import static greencity.dto.genericresponse.GenericResponseDto.buildGenericResponseDto;
+
+@Controller
+@AllArgsConstructor
```

Comment:
Can be changed to @Requiredargscontructor

Reply:
Done